python/tvm/auto_scheduler/compute_dag.py

```diff
@@ -72,7 +72,7 @@ def get_init_state(self):
         """
         return State(self.init_state, self)
 
-    def apply_steps_from_state(self, state):
+    def apply_steps_from_state(self, state, layout_rewrite=False):
```

Comment:
Add layout_rewrite to the doc string below.

Reply:
done